src/scanner.h

```diff
@@ -258,7 +258,7 @@ typedef struct {
 
 EXPORT_INLINE int IsIdent(char c)
 {
-    return IsAlpha(c) || c == '_' || c == '@';
```

Comment:
Unless your plan is to remove IsAlpha (which doesn't seem to be part of this PR?), I'd prefer we consistently use IsAlpha. Unless you have another plan?

Reply:
Longterm I'd like to remove `IsAlpha` simply because it seems pointless to "wrap" a perfectly fine ISO C function. But this PR is not meant to do that; I restored the code, but moved `IsIdent` to `sysstr.h`, where it seems to fit in better.